project/apps/matching-service/src/main.ts

```diff
@@ -1,23 +1,39 @@
 import { NestFactory } from '@nestjs/core';
 import { MatchingModule } from './matching.module';
 import { Transport, MicroserviceOptions } from '@nestjs/microservices';
+import { ConfigService } from '@nestjs/config';
+import { Module } from '@nestjs/common';
+
+@Module({
+  providers: [ConfigService],
+  exports: [ConfigService],
+})
+class BootstrapConfigModule {}
```

Comment:
this is needed to inject configservice into main, referenced from [here](https://artosalminen.github.io/posts/how-to-use-config-service-in-nestjs-bootstrap/)